modules/thermo-cycler/thermo-cycler-arduino/eeprom.h

```diff
@@ -0,0 +1,32 @@
+#ifndef eeprom_h
+#define eeprom_h
+
+#include "Arduino.h"
+#include <Wire.h>
+
+#define EEPROM_ADDR     0x52  // TODO: make this 0x52 for thermocycler
+#define MAX_IN_NUM_LEN  17  // TCV0120190903Annn
+#define BUFFER_BYTES    10
+#define SERIAL_LOC      10
+#define MODEL_LOC       SERIAL_LOC + MAX_IN_NUM_LEN + BUFFER_BYTES
```

Comment:
We should define the `MODEL_LOC` as an absolute address rather than as a derivation of the `SERIAL_LOC`, for a couple reasons:

1. If we define it as an absolute location, you can see what it is at a glance by looking at this file
2. When we define it as a derived location, we can accidentally change it by changing one of the priors, and break back compatibility for reading serials. 

In fact, I'd recommend something like this:

```c++
#define SERIAL_LOC 10
#define MAX_IN_NUM_LEN 17
#define MODEL_LOC  30

static_assert(MODEL_LOC + MAX_IN_NUM_LEN < SERIAL_LOC);
```

so you'll get a compiler error if it fails. And this should ideally be in a header file shared with the eeprom writer class.